ExtensionPack/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# C/C++ Extension Pack for Visual Studio Code Change Log
+
+
+## Version 0.1.0: September 1, 2020
+### List of extensions:
+* C/C++ (Microsoft)
+* C/C++ Themes (Microsoft)
+* Cmake Tools (Microsoft)
```

Comment:
CMake

Reply:
We can remove "(Microsoft)" from the names